tests/api/selfcheck/test_selfcheck.py

```diff
@@ -190,10 +191,57 @@ def test_patron_information(client, librarian_martigny,
         )
     )
     assert res.status_code == 200
+    # create validated request
+    circ_params = {
+        'item_pid': item3_lib_martigny.pid,
+        'patron_pid': selfcheck_patron_martigny.pid,
+        'pickup_location_pid': loc_public_martigny.pid,
+        'transaction_library_pid': lib_martigny.pid,
+        'transaction_user_pid': librarian_martigny.pid
+    }
+    res, data = postdata(
```

Comment:
Probably you need to do a cancel request to clean this test.

Reply:
yes of course.